src/less/widgets/widgets-general.less

```diff
@@ -38,6 +38,12 @@
     color: @text;
   }
 
+
+  .pf-widget-footer {
+    font-size: 12px;
+    text-align: left;
```

Comment:
Idk if this would be a hassle, but I noticed on all widgets this is left aligned. Some widgets like modal forms have center aligned text by default. Would be great to match the text alignment with the text above.